docs/changes/coordinates/16135.feature.rst

```diff
@@ -0,0 +1,2 @@
+Added a fast path for converting ``PhysicsSphericalRepresentation`` to
```

Comment:
Are performance improvements normally considered features?

Reply:
Personally don't think a changelog entry is required.

Reply:
Happy to remove. I was on the fence.

Reply:
Done.